libs/frontend/feature-build-event/src/lib/use-build-event.ts

```diff
@@ -342,7 +344,12 @@ export const useBuildEvent = () => {
       const resizeCmpRect = resizeCmp.getBoundingClientRect()
       const parentRect = (resizeCmp.parentElement as HTMLElement).getBoundingClientRect()
 
-      const startHeight = initialHeightProp || `${resizeCmpRect.height}px`
+      let startHeight = initialHeightProp || `${resizeCmpRect.height}px`
+      if (!isLengthValue(startHeight)) {
+        // Use the current height (px) of the resized component as the inital height
+        startHeight = `${resizeCmpRect.height}px`
+      }
```

Comment:
This is to fix a bug when the resize happens when the width/height of a component is not a length value (i.e. `auto`, `initial`, `inherit`, etc); that'll cause the width/height to be something like `0auto` or `NaNinitial` during the process.

Reply:
👍